docs: DOC-1840: Additional Bug Fixes for 4.6.b Release Notes (#6534)
spectrocloud/librarium: docs/docs-content/release-notes/release-notes.md

#### Bug Fixes

- Fixed an issue where tags that contained spaces prevented [AWS](../clusters/public-cloud/aws/aws.md) clusters from being deployed via API. 
- Fixed an issue where simultaneous updates to [EKS](../clusters/public-cloud/aws/eks.md) logging and VPC config caused reconciliation failures due to API limitations.
- Fixed an issue in [MAAS](../clusters/data-center/maas/maas.md) clusters where only the first node pool got repaved when a full cluster repave was expected.

> - Fixed a UI discrepancy where the worker node count on the cluster **Review** page always displayed `1` for clusters with autoscaler enabled. This did not affect cluster functionality.

Author: ```suggestion
- Fixed a UI discrepancy where the worker node count on the cluster **Review** page displayed the **Number of nodes in pool** instead of the **Minimum size** and **Maximum size** for clusters with autoscaler enabled. This did not affect cluster functionality.
```

Updated per Petronela.

- Fixed an issue that caused repeated reconciliation errors when deploying an [EKS](../clusters/public-cloud/aws/eks.md) cluster with private cluster endpoint access. This did not affect cluster functionality.
- Fixed an issue where nil pointer dereferences caused a panic and prevented [EKS](../clusters/public-cloud/aws/eks.md) clusters from being deployed.

> 

Author: ```suggestion
- Fixed an issue during cluster setup where selecting **Copy from Control Plane Pool** would reset certain worker pool configurations, such as autoscaler. Copied changes are now restricted to cloud configurations.

```

#### Improvements

- Palette and VerteX emails have been redesigned to ensure consistency and improve accessibility. The updates have been
  applied to sign-up, login, password reset, and billing update emails.

> - The existing **Enable Autoscaler** setting is now honored when copying configurations from a control plane or worker pool.

Author: ```suggestion
```
Rewriting and moving to bug fixes per Petronela.

  [Welcome to the fold: meet the new Spectro Cloud brand](https://www.spectrocloud.com/blog/meet-the-new-spectro-cloud-brand)
  blog post to learn more.

#### Bug Fixes

- Fixed an issue that prevented DNS configuration changes made using the Terminal User Interface (TUI) from being
  applied without restarting the `CoreDNS` deployment.

> - Fixed an issue where password updates were delayed on appliance mode [Edge](../clusters/edge/edge.md) hosts

Author: ```suggestion
- Fixed an issue where password updates were delayed on appliance mode [Edge](../clusters/edge/edge.md) hosts.
```
